dotcom-rendering/src/components/Discussion/Comments.tsx

```diff
@@ -96,6 +99,14 @@ const writeMutes = (mutes: string[]) => {
 	storage.local.set('gu.prefs.discussion.mutes', mutes);
 };
 
+/**
+ * Dispatches a custom event which is handled by @guardian/commercial
+ *
+ * @see https://github.com/guardian/commercial/blob/9175e572a5a81e334208775c64b7b479dd544c6c/src/insert/comments-expanded-advert.ts#L197
```

Comment:
This line identifier will become out of date pretty quickly, as I'm making currently making changes to the file. I would remove this line referencing the file from this comment, in case the file is renamed or moved. It should be easy enough to find where this custom event is listened to in Commercial by searching for `comments-state-change`.

Reply:
Yep fair point, I can remove it.

Reply:
Oh really, that's interesting. Cheers Max